homeassistant/components/devolo_home_network/manifest.json

```diff
@@ -8,7 +8,7 @@
   "iot_class": "local_polling",
   "loggers": ["devolo_plc_api"],
   "quality_scale": "platinum",
-  "requirements": ["devolo-plc-api==1.4.0"],
+  "requirements": ["devolo-plc-api==1.4.1"],
```

Comment:
Why do we have a timeout both here and in the library? Which one will trigger first and what will happen if the library times out?

Reply:
If the library times out, a DeviceUnavailable excetion will be raised and processed by HA.

Reply:
Ok. What will happen if the library times out?

Reply:
The DeviceUnavailable excetion is basically the issue, that is sometimes preventing the integration from setting up. But the answer actually comes, it just takes longer than expected.